spacy/cli/_util.py

```diff
@@ -131,6 +132,14 @@ def _parse_override(value: Any) -> Any:
     except ValueError:
         return str(value)
 
+def _handle_renamed_language_codes(lang: Optional[str]) -> None:
+    # Throw error for renamed language codes in v4
+    if lang in RENAMED_LANGUAGE_CODES:
+        msg.fail(
+            title="Renamed language code",
+            text=f"Language code '{lang}' was replaced with '{RENAMED_LANGUAGE_CODES[lang]}' in spaCy v4. Update the language code from '{lang}' to '{RENAMED_LANGUAGE_CODES[lang]}'.",
```

Comment:
Can you make sure to run `black` on all the code files?

Reply:
Weird, isn't this `black` formatted? I double-checked, and it always formats to this. Maybe something is configured wrong on my end?

Reply:
I'm pretty sure it isn't correctly formatted, no, as there should be 2 blank lines (not 1) before and after the function?

Reply:
Ah yes, you're right! Pushed the change and made sure that `black` is run on all the other 29 files. I wasn't paying attention to the trailing spaces but only looked at the code block, sorry!